src/key_io.cpp

```diff
@@ -190,7 +190,7 @@ CTxDestination DecodeDestination(const std::string& str, const CChainParams& par
                 return CNoDestination();
             }
 
-            return WitnessUnknown{version, data};
+            return WitnessUnknown{static_cast<unsigned int>(version), data};
```

Comment:
in https://github.com/bitcoin/bitcoin/pull/28246/commits/d33fa1fae66635992e6a0a980e7d16a9ebe4c852:

Is there a reason to do this here, as opposed to making an overloaded constructor for `WitnessUnknown`, like:

```
WitnessUnknown(int version, std::vector<unsigned char> program)
        : m_version(static_cast<unsigned int>(version)), m_program(std::move(program)) {}
```

that way, anything that creates a `WitnessUnknown` can just be `WitnessUnknown{version, program}`

Reply:
Done